pandas/tests/groupby/test_categorical.py

```diff
@@ -1380,3 +1380,13 @@ def test_groupby_agg_non_numeric():
 
     result = df.groupby([1, 2, 1]).nunique()
     tm.assert_frame_equal(result, expected)
+
+
+def test_groupy_first_returns_categorical():
+    # GH 28641: Issue mentioned in first comment.
```

Comment:
this comment here after the colon is redundant, can you give something more helpful

Reply:
Done